src/pages/details/components/priceList/priceListModal.tsx

```diff
@@ -74,12 +76,17 @@ class PriceListModalBase extends React.Component<Props> {
     }
 
     const priceListProvider = providers.data.find(
-      p => p.uuid === getSourceUUID(item)
+      (p) => p.uuid === getSourceUUID(item)
     );
     const priceListRates =
       priceListProvider && priceList[priceListProvider.uuid];
     return priceListRates ? (
-      <PriceListTable t={t} rates={priceListRates} />
+      <>
+        <PriceListTable t={t} rates={priceListRates} />
+        <Link to={`cost-models/${priceListRates.uuid}`}>
+          {t('details.price_list.modal.cost_model_link')}
+        </Link>
+      </>
```

Comment:
Perhaps you and Natalie discussed an alternate design, but don't believe we wanted to add a link in the existing price list table?

Instead of showing a price list, I believe the intention was that the kebab would take the user directly to the cost models details page.

> Desired Functionality: When a user is looking at a list of clusters or cloud accounts, we have an option in the row kebab to redirect the user to the assigned cost model's details page (if one is assigned to it).  If there is no associated source id with a cost model, we can have the option ... disabled.

Reply:
@dlabrecq I recall that you said it will be odd to have that option in the kebab because the user excepts a modal and not to be redirected to the cost model page.
@nlcwong do you remember what we decided to do here?